### DISCUSSION TITLE
Caltech CS122: Database System Implementation - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#200] [Announcements]
# Caltech CS122: Database System Implementation - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%95%B0%E6%8D%AE%E5%BA%93%E7%B3%BB%E7%BB%9F/CS122/

**Comment by dengyunsheng250** [comment]
mark
